libs/@guardian/source-react-components/src/@types/Space.ts

```diff
@@ -0,0 +1,3 @@
+import type { space } from '@guardian/source-foundations';
+
+export type Space = keyof typeof space;
```

Comment:
could we use the export from `@guardian/source-foundations` directly? i'm not sure what this gives us?

Reply:
This is replacing the component's own hardcoded list of spacing values:

```ts
export type InlineSpace = 1 | 2 | 3 | 4 | 5 | 6 | 9 | 12 | 24;
```

A couple of other components have a similar spacing type, but they're all missing values that have been subsequently added.

`@guardian/source-foundations` doesn't export a spacing type so you'd have to use `keyof typeof space` to get these. Foundations would be a better place for this though.

Perhaps I should add this to foundations and update all of the components to use it in a separate PR?

Reply:
sorry james, i was too hasty reviewing. i think this is the right approach, sorry for the noise